Makefile

```diff
@@ -27,9 +27,9 @@ workspace: aspects
 			-scheme XCHammer \
 			|  awk '$$1 == "BUILD_DIR" { print $$3 }'))
 	@mkdir -p "$(BUILD_DIR)"
-	@ditto "$(ASPECTDIR)" "$(BUILD_DIR)/Debug/"
+	@ditto "$(ASPECTDIR)" "$(BUILD_DIR)/Debug/TulsiGenerator.framework"
 	@ditto "$(ASSETDIR)" "$(BUILD_DIR)/Debug/$(ASSETDIR)"
-	@ditto "$(ASPECTDIR)" "$(BUILD_DIR)/Release/"
+	@ditto "$(ASPECTDIR)" "$(BUILD_DIR)/Release/TulsiGenerator.framework"
 	@ditto "$(ASSETDIR)" "$(BUILD_DIR)/Release/$(ASSETDIR)"
```

Comment:
I do not know how XCHammerAssets is used. Is this path correct? I haven't had any problems, but since the tulsi path was incorrect, I figured I'd ask about this one.

Reply:
Hey @kastiglione thanks for the PR! The `XCHammerAssets` is loaded from the main bundle. It looks like under `Xcode 10` they are generating `.frameworks` now for Xcode, so this change make sense. In order to preserve the behavior in Xcode 9, we may want to _add_ this `ditto`. Additionally, would you make the same change to the `Debug` directory?

PS: I'm really looking forward to getting onto Bazel and XCHammer for XCHammer 🎉

Reply:
> looks like under Xcode 10 they are generating .frameworks now for Xcode

they who?

Reply:
This change is made to both Release and Debug directories. GitHub isn't showing the full context.

Reply:
What's xchammer's support policy? I suggest master support only the latest Xcode :D

Reply:
Prior to Xcode 10, it generated static libs, so this would break backwards compatibility for the _developers using xcode_ - it seems fine for now

<img width="1484" alt="spmbuildingframeworks" src="https://user-images.githubusercontent.com/1245820/47125813-57562680-d23a-11e8-83b1-a1fc981ec4d5.png">

Reply:
I don't fully understand. You're saying this would break developers using Xcode 9?

Reply:
Exactly - under Xcode 9, SPM seemed to use static libs in the SPM Xcode projects. That won’t work with this ( that’s ok! ). 

The corresponding Tulsi NSBundle code loads resources from the adjacent directory to the containing binary: for frameworks, in the framework, and for static libs in the main bundle.. 

Since the issue is a SPM, Xcode, XCHammer development workflow issue, it wouldn’t impact the users of  XCHammer gen’d projects.on Xcode 9.